mh/equipment_utils.py

```diff
@@ -0,0 +1,1566 @@
+    QurupecoMail = 16
+    QurupecoVest = 17
+    LudrothMail = 18
+    LudrothVest = 19
+    SteelMail = 20
+    BaggiMail = 21
+    BaggiVest = 22
+    BarrothMail = 23
+    BarrothVest = 24
+    GobulMail = 25
+    GobulVest = 26
+    RangersGirdle = 27
+    ArchersGirdle = 28
+    GuildBardBolero = 29
+    GuildBardVest = 30
+    ScholarsBlouse = 31
+    ScholarsVest = 32
+    RathianMail = 33
+    RathianVest = 34
+    GigginoxMail = 35
+    GigginoxVest = 36
+    IngotMail = 37
+    IngotVest = 38
+    LagiacrusMail = 39
+    LagiacrusVest = 40
+    RathalosMail = 41
+    RathalosVest = 42
+    BariothMail = 43
+    BariothVest = 44
+    DiablosMail = 45
+    DiablosVest = 46
+    UragaanMail = 47
+    UragaanVest = 48
+    AgnaktorMail = 49
+    AgnaktorVest = 50
+    HeliosMail = 51
+    HeliosVest = 52
+    SeleneMail = 53
+    SeleneVest = 54
+    YamatoMuneate = 55
+    MutsuMuneate = 56
+    HinataMuneate = 57
+    AmagiMuneate = 58
+    LeatherMailPlus = 59
+    LeatherVestPlus = 60
+    ChainmailArmorPlus = 61
+    ChainmailVestPlus = 62
+    LocLacShawlPlus = 63
+    LocLacVestPlus = 64
+    HuntersMailPlus = 65
+    HuntersVestPlus = 66
+    BoneMailPlus = 67
+    BoneVestPlus = 68
+    AlloyMailPlus = 69
+    AlloyVestPlus = 70
+    JaggiMailPlus = 71
+    JaggiVestPlus = 72
+    BnahabraMailPlus = 73
+    BnahabraVestPlus = 74
+    RhenoplosMailPlus = 75
+    RhenoplosVestPlus = 76
+    QurupecoMailPlus = 77
+    QurupecoVestPlus = 78
+    LudrothMailPlus = 79
+    LudrothVestPlus = 80
+    SteelMailPlus = 81
+    BaggiMailPlus = 82
+    BaggiVestPlus = 83
+    BarrothMailPlus = 84
+    BarrothVestPlus = 85
+    GobulMailPlus = 86
+    GobulVestPlus = 87
+    GuildBardBoleroPlus = 88
+    GuildBardVestPlus = 89
+    ScholarsBlousePlus = 90
+    ScholarsVestPlus = 91
+    RathianMailPlus = 92
+    RathianVestPlus = 93
+    GigginoxMailPlus = 94
+    GigginoxVestPlus = 95
+    IngotMailPlus = 96
+    IngotVestPlus = 97
+    DoberMail = 98
+    DoberVest = 99
+    LagiacrusMailPlus = 100
+    LagiacrusVestPlus = 101
+    RathalosMailPlus = 102
+    RathalosVestPlus = 103
+    BariothMailPlus = 104
+    BariothVestPlus = 105
+    DiablosMailPlus = 106
+    DiablosVestPlus = 107
+    UragaanMailPlus = 108
+    UragaanVestPlus = 109
+    AgnaktorMailPlus = 110
+    AgnaktorVestPlus = 111
+    VangisMail = 112
+    VangisVest = 113
+    DamascusMail = 114
+    DamascusVest = 115
+    HeliosMailPlus = 116
+    HeliosVestPlus = 117
+    SeleneMailPlus = 118
+    SeleneVestPlus = 119
+    YamatoMuneatePlus = 120
+    MutsuMuneatePlus = 121
+    HinataMuneatePlus = 122
+    AmagiMuneatePlus = 123
+    EscadoraSoul = 124
+    EscadoraSeele = 125
+    ChakraNecklace = 126
+    HellHunterJacket = 127
+    HellHunterLeather = 128
+    SoulHunterJacket = 129
+    SoulHunterLeather = 130
+    err = 131
+    err = 132
+    err = 133
+    err = 134
+    err = 135
+    err = 136
+    err = 137
+    err = 138
+    err = 139
+    err = 140
```

Comment:
These fields should have unique names. Feel free to rename them `ERR131` and so on. Ditto for all the other occurrences below.